libhb/decsrtsub.c

```diff
@@ -68,27 +68,27 @@ read_time_from_string( const char* timeString, struct start_and_end *result )
 {
     // for ex. 00:00:15,248 --> 00:00:16,545
 
-    long houres1, minutes1, seconds1, milliseconds1,
-         houres2, minutes2, seconds2, milliseconds2;
```

Comment:
I can't think of any good reason for this. I know it parallels `minutes`, but it doesn't parallel `seconds`...